libs/blocks/graybox/mobileFrames.js

```diff
@@ -0,0 +1,5 @@
32ZGIMUjifbNbEaE3xYQYzuEO0RHHHcVpjAg8GqNoQTVeWwwLFv8sfvK0MVjW1xuF4NZgbbsvWc/G6SBSrPYB279k1J1Zz8Nn45MiMBF3U5xpjHCZWOYY545kFqRhOX66oK3HVN1e5YAS4Y6B5+uRQm9A/flIYv3RKOPPJiHD+swnh499N4gtfTMAzPvspedNeHsanJp0wcowHJnpw3+lu3DfSDQeD4t5THbhnqB139rdgsOy7BloVgPM6Iay1pxE3dzfA4cu4vbeZXJvItQm3EWg92YhbSBwA3NPwniaaVoaWrgZSvlRer5gXnjuczfS+9eQJGtdBoJWkh/Cu/jYCraT+ZqleT32AE1u+XCbcQ7yMhxxG8SH3tD4CEN/We4KpX0zf4WxRQMwLZTUo6eTy2uV83Y00rrOvVeQDoVNOB88dDO5yHSJPxXDIB9BG8zTT9C0Sn0U8mmk4LRPySOUA7wD/lpMi39t7xTrgHvLI9bL0QL1ifS1UtjQdhIFc+lu85EFpgmeQh0SHSEMD5YkCEq3tUp1UhlJaz73IHzkdyLR7sE3JS2XW3aDU6yU/iWalnt4mL5l3SvjJ7VGRjw+/WqQyKc8JrW1SvIy3AqS87oE2qY2foLhCHaKsWhQ6KM5SO5XbXHPncZqvlT5L7VCKo22Ntv1mKc7TdjtA7r0engFvYMl576lO2kcHRnvw0EQv7bsj5/ppX4ZJDpOfjgoffzmJL315Vjh/lRhyl4dhVz8BlPTJoXaqqBvaa6gr5FB1HlHWDpyea8GwkZ7ekoTV+nis0cdf0JnUlvQc6+NrnlyJ4rUxiOgcZHOYeUu6gXfkmpEpTQOH0Hz7ByNv1i86IQI58sxo/pL7kdGigymGKrUxmn/7nbdQVOJHKDJxH9q/P3JOjObAutikiAQCrfG6qD9o9HGYvBlxil2LrZmivxxcLYUVWdOltQXuivoSFxHWVH1b1dSJ7qOu5p7jro6BJnf3UIv75HCb2zna6e4f73YPTJx0D032uslnlHv4XJ977OKg+/SFfvf5z8bdH//unJs0Cnq9RK+T7gufT5D4AQpjlwbd45eGaPpR8nzm8jCBEff4x8M0fIzAxMenaBjckxcIDSeNi8JZ8nxGgskroxTo/aen3eRFQ/KK6aAMuEI8hE9eYQDSMuWQxuw+d3WMwriUT46D53G2LCXPiFKueB2l5cA9lCPGi/nPAZ6fjop0SfkAT4r31VGaXqQF8o7S9HJZMl0yKM8k/jzUA2FXRxXcoD4IgzgPn8Q8Zz4R88v1KWVekeqU8oEcAUS+jtJ7OQ/L+wufTYh4++AJeWQ4J9EF+MpymqT0n5LkeVqp4yJpJ2dkOUt4yXyFK6QR8WfKBboujyjxZyXZsvwS5S7iSdMALqSOc1fGvGVHgJU/1ANygnZL85L0sixlkPPJ8oL8k5+K5ctyUtrbZbFNQNlQrswL2v4vnaLtfZT0JdrmaF8YFPl9eViMvzjkHiH9BgD6D6Qf/1jMQxQv7YvD5/vodehsj3vwTI97AID0V+i70Ic7B5rd4J9u7jnmbuiqdR1tqZqqaSxzlR0tIko6311Ynj2dW+rA9lwTtpAveNhzSW/VYPLFj1Nsun81mLXHjFbdwdzCzLVoFUIx6giUlmWhPuwkYmUTg5IzmpPR1vD1KCE5CmXmpQUUtr9fFFHi6dlWtP2376DX33kFJSbHggtFFaeL4Lfv2IEi4/dRZb4/cg+KiY+6N0Z98ImIuH27Y5MOFsZqIoYI47+M00a4NMZYnGxJxOByAWGBYk/LNWNHngVnEIs9pyQd5x7OwHmHMwlkgTuGKPwcXFyZh0uq8mApPy6rLcJtzkZilRELg0DXoAjdg2DlNeLyumJ8qCqfQIEIR8R78glG/fAApdUFsABJeYYyIV9ZTZEIteKVpiFX8hWB4cQNuLJp2HAZ4EglX6DlUZDq8pOGApTPlnvUUz7Ew72Mp5i+2IO3VO5hcn+Y4E3Ts+XJIJdXJ8cXK2kr6or90iKXWXnskFhfrYd+mU8yPjKdct0K3iy+DD+UeuqYPFI6Wi+VW5EPP334KOUVy5bqYeuV6ZDkqfCQlZfMY4l2ud7Keg/NSjoFNxE8cR7wbQszeOJVTqGnnTCyZcvw8MwXiml+uW+w7VeuV6a7jKnD014KaX+BPkGvVeIV+psM9PlIHgX5vqgyFxeRvlkIfZRAflkWXakNg4y5xQ6cXZgGJ9SQvm3FqVkmDKd6WdPBV63FyWY11hjicFxSxJ9i1RGfJ+iiTxHdkGu06d4j8euGTg+GvLntNZRsSkIWuxFt2/k8StDFEAvbpoJl7DaHgZ4cZs8yByztr/sjb0R06HAZyi5woFhNJCLWNlHkkXycJoo7EEUs8viP0IGYvShGsx/tjd6JXnw5/K7fbP/10v1Rezb8dv8H73649/24fREfOiLj9pdGJew/Hp1woJ3k7UvQxjiJkJyJ2linWp/g1JkTFUi2qCmkpGqdhWU5zkNHCpwllfkECug9QFF5rtOSrncaSRq9NYmk19CrCBoKBpvWabLrnEZbEi3LKD2bUnX0Ge7hKoP8LKfzCpPu5fzKvQ+IZXnKNaV61wlle5Wf6r8MORzqNduTaZkGiQ4AoI3FezacaD6rVqn/WmC0+ZTH8AKuUD/cU76mimFefJEA+C2X5U0ryxudV31ymEHKO4OncPWih8jeJuFl1dJ7uUyDNUnBmeVLCit3u6d+EU+ZtiRKEy1Lwh/CZPBtE96yEPPIfGLbU4r0DLixZbFtQsRP51UfpYWRvdzeZb4r8lFwEdu/kWmzSj1MXSwenn4j5iUKV+yHVjWNp88mKYzca1MSnRpjvFOti3NCPyb92RmdcLCP9PGufRG763ft21m09+CHKbv37YjevW/nO3sO7HrqzW2/XPTa66/cuTdyB1HOUWBRI4MlCXWcbEdvbHtVBcvY07Nsqhj1AYQFTHVPVn76ba0b/x8snrKjrB/gEQAAAABJRU5ErkJggg==';
```

Comment:
How was this generated, should we leave some instructions on how to regenerate this if needed as comment above?

Reply:
I modified an image of an iphone to turn it into a "frame", then took the png to a png -> data:image converter.  I don't foresee needing to update this except for possibly upgrading the look.